lib/globber.js

```diff
@@ -1,29 +1,35 @@
 'use strict';
 const glob = require('glob');
 const Rx = require('rx');
+const fs = require('fs');
 
 function globber(paths) {
   const files = new Rx.Subject();
   files.fileEvents = [];
 
-  const Glob = glob.Glob;
   let doneCount = 0;
 
-  paths.forEach(function (path) {
-    const fileEvents = new Glob(path);
+  function globFiles(path) {
+    const fileEvents = new glob.Glob(path);
 
     fileEvents.on('match', function (file) {
-      files.onNext(file);
+      if (fs.statSync(file).isFile()) {
+        files.onNext(file);
+      } else {
+        globFiles(file + '/*');
+      }
```

Comment:
plus: this is mostly to handle posix expanding paths.

a `'**/*'` is well handled by glob, when not expanded